bluetooth/src/androidLibMain/kotlin/device/DeviceConnectionManager.kt

```diff
@@ -136,6 +136,7 @@ internal actual class DeviceConnectionManager(
     }
     private var lastKnownState = BluetoothProfile.STATE_DISCONNECTED
 
+    @SuppressLint("MissingPermission")
```

Comment:
Was this not solved by adding the connect permission I saw elsewhere?

Reply:
Nope, still getting a warning